bigchaindb/core.py

```diff
@@ -131,23 +129,30 @@ def reassign_transaction(self, transaction, durability='hard'):
             # There is no other node to assign to
             new_assignee = self.me
 
-        response = self.connection.run(
-                r.table('backlog')
-                .get(transaction['id'])
-                .update({'assignee': new_assignee, 'assignment_timestamp': time()},
-                        durability=durability))
-        return response
+        return self.backend.update_transaction(
```

Comment:
MongoDB has read and [write concerns](https://docs.mongodb.com/manual/reference/write-concern/) where one can specify whether to return immediately, wait until written to disk, or wait until written to a majority of nodes.  This can definitely be set globally (collection or db-wide...or both?) but I'm not sure if it can be set on a per query basis.